Documentation/ipvlan.md

```diff
@@ -38,3 +38,40 @@ Because all ipvlan interfaces share the MAC address with the host interface, DHC
 Therefore the container will not be able to reach the host via `ipvlan` interface.
 Be sure to also have container join a network that provides connectivity to the host (e.g. `ptp`).
 * A single master interface can not be enslaved by both `macvlan` and `ipvlan`.
+
+### Using ipvlan and dhcp
+There are two considerations when using "ipvlan" plugin together with "dhcp" IPAM plugin.
```

Comment:
There are two important considerations when using the "ipvlan" plugin together with the "dhcp" IPAM plugin.